assets/js/modules/idea-hub/components/dashboard/DashboardIdeasWidget/Pagination.js

```diff
@@ -60,19 +62,47 @@ const Pagination = ( { tab } ) => {
 		return 0;
 	} );
 
+	const trackPage = useCallback(
+		( direction ) => {
+			const eventMap = {
+				forward: {
+					'new-ideas': 'new_page_advance',
+					'save-ideas': 'saved_page_advance',
+					'draft-ideas': 'draft_page_advance',
+				},
+				back: {
+					'new-ideas': 'new_page_return',
+					'save-ideas': 'saved_page_return',
+					'draft-ideas': 'draft_page_return',
+				},
+			};
+			const event = eventMap[ direction ][ tab ];
+
+			trackEvent( IDEA_HUB_GA_CATEGORY_WIDGET, event, page );
```

Comment:
Let's probably add a check to make sure that `event` is not empty.

Reply:
Good idea